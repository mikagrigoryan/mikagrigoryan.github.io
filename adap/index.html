<!DOCTYPE html>
<html lang="en">
<head>
	<link rel="stylesheet" type="text/css" href="css/style.css">
	<title>Golden</title>
</head>
<body>
	<div id='wrapper'>
		<img src="images/logo.png" id="logo">
		<ul>
			<a href="#"><li>Home</li></a>
			<a href="#"><li>Services</li></a>
			<a href="#"><li>Portfolio</li></a>
			<a href="#"><li>About</li></a>
			<a href="#"><li>Contact</li></a>
		</ul>
		<!-- Welcome to our studio -->
		<span id="welcome">welcome to our studio</span>
		<!-- It's nice to meet you -->
		<span id="nice">it's nice to meet you</span>
		<!-- Button -->
		<a href="#"><img src="images/button.png" id="button"></a>
		<img src="images/first-bg.png" id="first-bg">
	</div>
</body>
</html>