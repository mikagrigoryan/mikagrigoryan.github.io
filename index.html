<!DOCTYPE html>
<html>
<head>
	<title>MoGo</title>
	<link rel="stylesheet" type="text/css" href="css/style.css">
	<!--
		Importing Fonts
	-->
	<link href="https://fonts.googleapis.com/css?family=Montserrat:400,700&display=swap" rel="stylesheet">
	<link href="https://fonts.googleapis.com/css?family=Kaushan+Script&display=swap" rel="stylesheet">
	<link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">

</head>
<body>
<header>
	<span id='logo'>MoGo</span>
	<span id='nav1'><a class='nav' href="#" style="color:#fce38a;">ABOUT</a></span>
	<a class="navnclr" href="#" style="margin-left: 1048px;">SERVICE</a>
	<a class="navnclr" href="#" style="margin-left:1165px;">WORK</a>
	<a class="navnclr" href="#" style="margin-left:1269px;">BLOG</a>
	<a class="navnclr" href="#" style="margin-left:1365px;">CONTACT</a>
	<a href="#"><img src="image/cart.png" id="cart"></a>
	<a href="#"><img src="image/lupa.png" style="width: 18px;height: 18px;position: absolute;margin-top: 36px;margin-left:1544px;"></a>
	<img src="image/underline.png" id="undrln">
	<span id="ct">Creative Template</span>
	<label id="welcome_label">WELCOME <br>TO MOGO</label>
	<img src="image/underline2.png" id="undrln2">
	<a href="#"><img id="button" src="image/button.png"></a>
	<img src="image/line1.png" id="line1">
	<img src="image/line2.png" id="line2">
	<img src="image/line2.png" id="line3">
	<img src="image/line2.png" id="line4">
	<a href="#" id="intro"><span class="number">01</span> <span class="word">Intro</span></a>
	<a href="#" id="two"><span class="number">02</span> <span class="word">Work</span> </a>
	<a href="#" id="three"><span class="number">03</span> <span class="word">About</span></a>
	<a href="#" id="four"><span class="number">04</span> <span class="word">Contacts</span></a>
	<img src="image/intro.jpg" id="headerimg">
</header>

<section id="content1">
	<div id="main">
		<span id="what_we_do">What we do</span>
		<span id="story">Story about us</span>
		<img src="image/line_red.png" id="line_red">
		<span id="text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim <br> veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. </span>
		<img src="image/photo-1.png" id="photo1">
		<img src="image/photo-2.png" id="photo2">
		<img src="image/photo-3.png" id="photo3">
	</div>
</section>

<section id="content2">
	<div id="main2">
		<img src="image/vertical_line.png" id="vertical_line">
		<img src="image/vertical_line.png" id="vertical_line2">
		<img src="image/vertical_line.png" id="vertical_line3">
		<img src="image/vertical_line.png" id="vertical_line4">
		<img src="image/vertical_line.png" id="vertical_line5">
		<img src="image/vertical_line.png" id="vertical_line6">

		<span id="number1">42</span>
		<span id="number2">123</span>
		<span id="number3">15</span>
		<span id="number4">99</span>
		<span id="number5">24</span>

		<span id="word1">Web design projects</span>
		<span id="word2">Happy client</span>
		<span id="word3">award winner</span>
		<span id="word4">cup of coffee</span>
		<span id="word5">members</span>
	</div>
</section>
<section id="content3">
	<div id="main3">
		<span id="www">We work with</span>
		<span id="as">AMAZING SERVICES</span>
		<img src="image/line_red.png" id="line_red2">
		<img src="image/alarm_1.png" class="icons" style="margin-top:290px; margin-left: 390px;">
		<img src="image/graph_2.png" class="icons" style="margin-top:289px;margin-left:  791px;">
		<img src="image/computer_3.png" class="icons" style="margin-top:290px;margin-left:1190px;">
		<img src="image/book_4.png" class="icons" style="margin-top:459px;margin-left:391px;">
		<img src="image/HOME_5.png" class="icons" style="margin-top:459px;margin-left:790px;">
		<img src="image/image_6.png" class="icons" style="margin-top:459px;margin-left:1190px;">
		<span class="text_big" style="margin-top:289px;margin-left:451px;">Photography</span>
		<span class="text_big" style="margin-top:289px;margin-left:850px;">Web design</span>
		<span class="text_big" style="margin-top:289px;margin-left:1250px;">Creativity</span>
		<span class="text_big" style="margin-top:459px;margin-left:450px;">Seo</span>
		<span class="text_big" style="margin-top:457px;margin-left:850px;">Css/Html</span>
		<span class="text_big" style="margin-top:459px;margin-left:1251px;">Digital</span>
		<span class="text_small" style="margin-top:319px; margin-left:450px;">Lorem ipsum dolor sit amet, consectetur <br> adipiscing elit, sed do eiusmod tempor.</span>
		<span class="text_small" style="margin-top:319px;margin-left:850px;">Lorem ipsum dolor sit amet, consectetur <br> adipiscing elit, sed do eiusmod tempor.</span>
		<span class="text_small" style="margin-top:319px;margin-left:1250px;">Dolor sit amet, consectetur adipiscing elit,<br> sed do eiusmod tempor.</span>
		<span class="text_small" style="margin-top:489px;margin-left:450px;">Ipsum dolor sit amet, consectetur<br> adipiscing elit, sed do eiusmod.</span>
		<span class="text_small" style="margin-top:489px;margin-left:850px;">Lorem dolor sit amet, consectetur<br> adipiscing elit, sed do tempor.</span>
		<span class="text_small" style="margin-top:489px;margin-left: 1250px;">Sit amet, consectetur adipiscing elit, sed <br> do eiusmod tempor.</span>
	</div>
</section>
<section id="content4">
	<div id="main4">
		<span id="foralldevices">For all devices</span>
		<span id="unique">Unique design</span>
		<img src="image/line_red.png" style="position: absolute;margin-top:220px;margin-left: 930px;">
		
		<img src="image/background_2.png" style="opacity: 0.2;z-index: -1; position: absolute;">
		<img src="image/ipad.png" style="position: absolute;margin-top:268px;margin-left: 695px; opacity: 1;z-index: 2;">
		<img src="image/iphone.png" style="position:absolute;opacity:1;z-index: 2;margin-top:465px;margin-left: 1057px;">
	</div>
</section>
<section id="content5">
	<div id="main5">
		<span id="srvc">Service</span>
		<span id="what_we_do2" style="	margin-top:155px;margin-left:854px;">What we do</span>
		<img src="image/line_red.png" style="margin-top:220px;margin-left: 930px;position: absolute;">
		<span id="txt">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim <br> veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. </span>
		<img src="image/photo.png" style="margin-top:400px;margin-left:360px">
	</div>
</section>

</body>
</html>