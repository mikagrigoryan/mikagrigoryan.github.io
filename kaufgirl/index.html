<!DOCTYPE html>
<html>
    <head>
        <title>Лиза</title>
        <link href="https://fonts.googleapis.com/css2?family=Lobster&display=swap" rel="stylesheet">
        <style>
            *{
                background: rgb(131,58,180);
                background: linear-gradient(90deg, rgba(131,58,180,1) 0%, rgba(253,29,29,1) 50%, rgba(252,176,69,1) 100%);
                color:white;
                font-family: 'Lobster',cursive;
                letter-spacing: 1px;
                text-align: center;
                
            }
            .all{
                background:none;
                max-width:600px;
                min-width:300px;
                margin-left:50vw;
                transform:translateX(-50%);
            }
            h1, p, h2{
                background:none;
            }
            #petrovimika{
                font-size:25px;
            }
        </style>
    </head>
    <body>
        <img url="https://image.freepik.com/free-vector/colorful-festive-balloons-design-vectors_53876-58079.jpg"/>
        <div class="all">
            <h1>Дорогая Лиз очбка!</h1>
            <h2>Поздравляем тебя с твоим 14-летием!</h2>
            <p id="one">Желаем, чтобы всё забытое плохое не всплыло вновь, а всё хорошее, что плавает по волнам, не пустилось на дно. Крепкого тебе здоровья, такого же крепкого, как терпение народа русского. Желаем новых знакомств, что будут греть твою душу до конца длинного и сложного пути, под названием "Жизнь". Желаем, чтобы все проблемы когда-нибудь решились.
                Желаем без проблем получить паспорт, которым ты будешь пользоваться чуть ли не каждый день (нет)</p>
            <p id="petrovimika">С любовью, Петров и Мика</p>
        </div>
    </body>
</html>