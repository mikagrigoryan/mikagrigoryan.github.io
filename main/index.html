<!DOCTYPE html>
<html>
<head>
	<title>Second Template</title>
	<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;800&display=swap" rel="stylesheet">
	<link rel="stylesheet" type="text/css" href="style/style.css">
	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
	<link href="https://fonts.googleapis.com/css2?family=Kaushan+Script&display=swap" rel="stylesheet">
	<link href="https://fonts.googleapis.com/css2?family=MuseoModerno:wght@600&display=swap" rel="stylesheet">
	<link href="https://fonts.googleapis.com/css2?family=Montserrat+Alternates&display=swap" rel="stylesheet">
	<link rel="stylesheet" href="https://unpkg.com/aos@next/dist/aos.css" />
	<link rel="stylesheet" href="https://unpkg.com/swiper/swiper-bundle.min.css">
</head>
<body>
	<script src="https://unpkg.com/aos@next/dist/aos.js"></script>
	<script src="https://unpkg.com/swiper/swiper-bundle.min.js"></script>
  	<script>
    	AOS.init();
  	</script>
	<div class="content">
		<header>
			<div class="nav">
				<ul>
					<li data-aos="fade-right" data-aos-duration="1000"><a onclick="document.getElementById('home').scrollIntoView()" data-text="Home">Home</a></li>
					<li data-aos="fade-left" data-aos-duration="1000"><a onclick="document.getElementById('about').scrollIntoView()" data-text="About Us">About Us</a></li>
					<li data-aos="fade-right" data-aos-duration="1000"><a onclick="document.getElementById('services').scrollIntoView()" data-text="Services">Services</a></li>
					<li data-aos="fade-left" data-aos-duration="1000"><a onclick="document.getElementById('contacts').scrollIntoView()" data-text="Contacts">Contacts</a></li>
				</ul>
			</div>
		</header>
		<div id="home">
			<div class="title">
				<span data-aos="fade-up">Home</span>
			</div>
			<div class="cards">
				<section>
					<h3>Card One</h3>
					<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
					tempor incididunt ut labore et dolore magna aliqua.Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
					tempor incididunt ut labore et dolore magna aliqua.Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
					tempor incididunt ut labore et dolore magna aliqua.</p>
				</section>
				<section>
					<h3>Card Two</h3>
					<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
					tempor incididunt ut labore et dolore magna aliqua.Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
					tempor incididunt ut labore et dolore magna aliqua.Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
					tempor incididunt ut labore et dolore magna aliqua.</p>
				</section>
				<section>
					<h3>Card Three</h3>
					<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
					tempor incididunt ut labore et dolore magna aliqua.Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
					tempor incididunt ut labore et dolore magna aliqua.Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
					tempor incididunt ut labore et dolore magna aliqua.</p>
				</section>
			</div>
		</div>
	<div id="about">
		<div class="title">
			<span style="color:white" data-aos="fade-in" data-aos-duration="1500">About</span>
		</div>
			<div class="carousel">
				<div class="carousel-item">
					<img src="img/img1.jpg" />
				</div>
				<div class="carousel-item">
					<img src="img/img2.jpg" />
				</div>
				<div class="carousel-item">
					<img src="img/img3.jpg" />
				</div>
				<div class="carousel-item">
					<img src="img/img4.jpg" />
				</div>
				<div class="carousel-item">
					<img src="img/img5.jpg" />
				</div>
			<script src="https://code.jquery.com/jquery-3.4.1.js"></script>
			<script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
			<script type="text/javascript">
				$(document).ready(function(){
				    $('.carousel').carousel();
				  });
			</script>
		</div>
			<span id="swipe">Swipe!</span>
			<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam consequat vulputate nunc, vel luctus ipsum dictum auctor. In imperdiet mauris metus, at fermentum eros egestas in. Integer vitae sem enim. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Nam lorem mi, aliquet eget pharetra sit amet, lobortis eu velit. Nullam condimentum, risus et vestibulum dapibus, sapien erat rutrum erat, et porttitor sem elit eu mauris. Praesent vel posuere nibh. Suspendisse potenti. Aenean ornare eleifend sapien, a vulputate libero congue eget. Integer pharetra, mi at interdum pulvinar, elit nibh faucibus dolor, at sollicitudin felis dolor eget neque. Duis vel viverra velit. Sed pulvinar risus nec finibus mattis. Sed ornare elit nisi, a sollicitudin metus blandit semper.</p>
	</div>
</div>
	<div id="services">
		<div class="title">
			<span data-aos="fade-up">Services</span>
		</div>
		<div class="swiper-container">
		    <div class="swiper-wrapper">
		      <div class="swiper-slide" style="background-image:url(./img/nature-1.jpg)"></div>
		      <div class="swiper-slide" style="background-image:url(./img/nature-2.jpg)"></div>
		      <div class="swiper-slide" style="background-image:url(./img/nature-3.jpg)"></div>
		      <div class="swiper-slide" style="background-image:url(./img/nature-4.jpg)"></div>
		      <div class="swiper-slide" style="background-image:url(./img/nature-5.jpg)"></div>
		    </div>
    <!-- Add Pagination -->
    		<div class="swiper-pagination"></div>
  		</div>
  		<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
  		tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
  		quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
  		consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
  		cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
  		proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
		<script>
		    var swiper = new Swiper('.swiper-container', {
		      effect: 'cube',
		      grabCursor: true,
		      cubeEffect: {
		        shadow: true,
		        slideShadows: true,
		        shadowOffset: 20,
		        shadowScale: 0.94,
		      },
		      pagination: {
		        el: '.swiper-pagination',
		      },
		    });
  		</script>
	</div>
	<div id="contacts">
		<span class="title" style="color:white;">Contact Us!</span>
		<div class="contact">
			<input type="text" id="name" placeholder="Name" />
			<input type="text" id="surname" placeholder="Surname" />
			<input type="text" id="email" placeholder="Email" />
			<textarea placeholder="Your Review" style="resize:none;" id="review"></textarea>
			<input type="submit" value="Submit!"/>
		</div>
	</div>
	<div id="copyrights">
		<p><b>Your Copyrights here</b> Migi company's website template.</p>
	</div>
</body>
</html>