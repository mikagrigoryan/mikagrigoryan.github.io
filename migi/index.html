<!DOCTYPE html>
<html>
<head>
	<title>Error 404</title>
	<link rel="stylesheet" type="text/css" href="style/style.css">
	<meta charset="utf-8">
	<meta content="ie=edge">
	<link href="https://fonts.googleapis.com/css2?family=MuseoModerno:wght@600&display=swap" rel="stylesheet">
	<link rel="stylesheet" href="https://unpkg.com/aos@next/dist/aos.css" />
</head>
<body>
	<script src="https://unpkg.com/aos@next/dist/aos.js"></script>
	<script>
    	AOS.init();
  	</script>
	<div id="main"> 
		<span data-aos="fade-left">404</span>
	</div>
	<div id="content">
		<span>Page not found!</span>
	</div>
	<div id="button">
		<a href="#"><span>Go Back!</span></a>
	</div>
</body>
</html>