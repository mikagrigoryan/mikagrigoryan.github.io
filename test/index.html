<!DOCTYPE html>
<html lang="ru" dir="ltr">
  <head>
    <meta charset="utf-8">
    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
    <title>Картошка chanel</title>
    <link rel="stylesheet" href="css/style.css" type="text/css"/>
  </head>
  <body>
      <a class="logo" href="https://www.youtube.com/channel/UCFj7uiOIdsNB3Jb42jQEkvA"><h1>Канал Dato</h1></a>
      <ul>
        <a class="nav" href="https://vk.com/id450645994"><li>Я ВКонтакте</li></a>
        <a class="nav" href="https://www.facebook.com/dato.merabyan"><li>Я в Фейсбуке</li></a>
        <a class="nav" href="https://www.instagram.com/datomerabyan/"><li>Я в Инстаграме</li></a>
      </ul>
      <hr/>
      <h2>Некоторые из моих видео: </h2>
      <iframe width="445" height="350"src="https://www.youtube.com/embed/yFMOOHRga6s" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
      <iframe width="445" height="350"src="https://www.youtube.com/embed/Zb-JuPiLSPU" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
      <iframe width="445" height="350"src="https://www.youtube.com/embed/A5-V-svd_Zo" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
      <iframe width="445" height="350" src="https://www.youtube.com/embed/pLTuL24OY7k" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
      <iframe width="445" height="350" src="https://www.youtube.com/embed/sHY1QV7q9v8" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
      <iframe width="445" height="350" src="https://www.youtube.com/embed/JbNkXewZfjc" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
      <h2>Описание: </h2>
      <h5>Всё о Brawl Stars, включая гайды, летсплеи, прикольные моменты и т.д</h5>
      <h5 style="color:green">Бро, подпишись, пожалуйста!</h5>
  </body>
</html>
