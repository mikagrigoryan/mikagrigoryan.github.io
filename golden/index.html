<!DOCTYPE html>
<html lang="en">
<head>
	<link rel="stylesheet" type="text/css" href="css/styles.css">
	<title>Golden</title>
</head>
<body>
	<div id='wrapper'>
		<img src="images/logo.png" id="logo">
		<ul>
			<a href="#"><li>Home</li></a>
			<a href="#"><li>Services</li></a>
			<a href="#"><li>Portfolio</li></a>
			<a href="#"><li>About</li></a>
			<a href="#"><li>Contact</li></a>
		</ul>
		<img src="images/first-bg.png" id="first-bg">
	</div>
</body>
</html>